services/time/src/lib.rs

```diff
@@ -35,381 +35,42 @@ extern crate serde_derive;
 #[macro_use]
 extern crate serde_json;
 
-use iron::{Handler, prelude::*};
-use router::Router;
-use chrono::{DateTime, Duration, TimeZone, Utc};
-
-use std::sync::{Arc, RwLock};
-
-use exonum::blockchain::{ApiContext, Blockchain, ExecutionError, ExecutionResult, Schema, Service,
-                         ServiceContext, Transaction, TransactionSet};
-use exonum::messages::{Message, RawTransaction};
-use exonum::encoding::serialize::json::reexport::Value;
-use exonum::storage::{Entry, Fork, ProofMapIndex, Snapshot};
-use exonum::crypto::{Hash, PublicKey};
+use exonum::api::Api;
+use exonum::blockchain::{ApiContext, ExecutionError, Service, ServiceContext, Transaction,
+                         TransactionSet};
+use exonum::crypto::Hash;
 use exonum::encoding;
+use exonum::encoding::serialize::json::reexport::Value;
 use exonum::helpers::fabric::{Context, ServiceFactory};
-use exonum::api::Api;
+use exonum::messages::RawTransaction;
+use exonum::storage::{Fork, Snapshot};
+
+use iron::Handler;
+use router::Router;
+use schema::TimeSchema;
+use time_provider::{SystemTimeProvider, TimeProvider};
+use transactions::*;
+
+/// Node API.
+pub mod api;
```

Comment:
According to our [conventions](https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md#conventions) `pub mod` goes before `use`.